modules/javafx.graphics/src/main/native-glass/gtk/glass_general.h

```diff
@@ -241,15 +241,13 @@ struct jni_exception: public std::exception {
     extern jmethodID jColorRgb; // javafx.scene.paint.Color#rgb(IIID)Ljavafx/scene/paint/Color;
 
 #ifdef VERBOSE
+#define LOG(msg, ...) { printf(msg, ##__VA_ARGS__); fflush(stdout); }
```

Comment:
This needs its empty/noop variant in the #else block as well, otherwise build will break on Release

Reply:
Fixed